test/test_config.c

```diff
@@ -0,0 +1,99 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include "axl.h"
+#include "axl_internal.h"
+
+#include "kvtree.h"
+#include "kvtree_util.h"
+
+int
+main(void) {
+    int rc;
+    char *state_file = NULL;
+    kvtree* axl_config_values = kvtree_new();
+
+    size_t old_axl_file_buf_size = axl_file_buf_size;
+    int old_axl_debug = axl_debug;
+    int old_axl_make_directories = axl_make_directories;
+    int old_axl_copy_metadata = axl_copy_metadata;
+
+    rc = AXL_Init(state_file);
```

Comment:
```suggestion
    rc = AXL_Init(NULL);
```
...and remove `state_file`.  `AXL_Init(state_file)` is deprecated.

Reply:
Done.